basic/map.cpp

```diff
@@ -129,6 +129,19 @@ void ARegionList::CreateNexusLevel(int level, int xSize, int ySize, char const *
 	}
 }
 
+// void ARegionList::CreateConstrainedSurfaceLevel(int level, int xSize, int ySize, char const *name, int contients,
```

Comment:
I am not sure why this is commended here

Reply:
It looks (from looking at the code and the history) that this was added by Valdis about 2 years ago for neworigins as an attempt to improve the map generation, and was added as a dummy/empty function to the other rulesets, then later removed from the game.h header and commented out everywhere.  This is (to me) prime example of code that should be just tossed since it's pretty obvious that an even better system has already replaced it.